### ISSUE TITLE
Check download / install funnel on brave.com for breaks (especially on macOS)

### ISSUE THREAD

**Issue by rmcfadden3** [brave/qa-resources#424]
#### Summary:

Per Slack convo with @stephendonner, we'd like to QA the download / install funnel on brave.com, from various OS, browser, and screen size combos (possibly even for different languages). Ideally focusing on macOS / Safari first.

Looking for any issues with placement or appearance (or disappearance) of the various download buttons, oddities with the top-level nav or download package, blank / incomplete pages, the download button not appearing on certain pages, the new download interstitial page, etc etc.

Basically anything that might hinder a user's ability to successfully see the download options, and complete the download / install.

CC @etridgely & @laurentiugruia

**Comment by LaurenWags** [comment]
Closing as stale, please advise if still required.
